ci-operator/config/open-cluster-management/multicloud-operators-foundation/open-cluster-management-multicloud-operators-foundation-main.yaml

```diff
@@ -3,6 +3,10 @@ base_images:
     name: ubi-minimal
     namespace: ocp
     tag: "8"
+  open-cluster-management_builder_go1.15-linux-amd64:
```

Comment:
You don't need this part. Base images define external images to use as the "from:" field in the "images:" stanza. For "images:" you only need "base".